packages/lexicon/README.md

```diff
@@ -0,0 +1,42 @@
+The lexicon (aka the OpenFunctionicon) is a central repositoty of key type and word definitions.
+
+It's a types repo and glossary at the same time.
+
+## Overview
+
+The OpenFunction stack is built on the concepts of Workflows, Runs, Jobs and Expressions (and more). Some of these terms can be used interchangable, or used differently in certain contexts.
+
+Here are the key concepts
+
+- An **Expression** is a string of Javascript (or Javascript-like code) written to be run in the CLI or Lightning.
+- A **Job** is an expression plus some metadata required to run it - typically an adaptor and credentials.
+  The terms Job and Expression are often used interchangeably.
+- A **Workflow** is a series of steps to be executed in sequence. Steps are usually Jobs (and so job and step are often used
+  interchangeably), but can be Triggers.
```

Comment:
i think we don't technical refer to triggers as steps. right now, all available "steps" are (javascript-based) jobs. in the future, we might introduce a "notification" step which behaves differently. or even a "loop over array" step which does something else entirely. but for now, all steps are jobs.